lms/djangoapps/mobile_api/users/tests.py

```diff
@@ -2,17 +2,35 @@
 Tests for users API
 """
 import datetime
+import ddt
+from django.conf import settings
 from django.utils import timezone
+from django.template import defaultfilters
+from mock import patch
+import pytz
 
 from xmodule.modulestore.tests.factories import ItemFactory, CourseFactory
 from student.models import CourseEnrollment
 from certificates.models import CertificateStatuses
 from certificates.tests.factories import GeneratedCertificateFactory
+from courseware.access_response import (
+    MilestoneError,
+    StartDateError,
+    VisibilityError,
+)
+from xmodule.course_module import DEFAULT_START_DATE
 
 from .. import errors
 from ..testutils import MobileAPITestCase, MobileAuthTestMixin, MobileAuthUserTestMixin, MobileCourseAccessTestMixin
 from .serializers import CourseEnrollmentSerializer
 
+from util.milestones_helpers import (
+    set_prerequisite_courses,
+    seed_milestone_relationship_types,
+)
+
+# pylint: disable=no-member
```

Comment:
ah, didn't realize all of them already had pylint disables, so i ended up moving it back to the individual lines